### PR TITLE
fix of paid section and posts page

### PR DESCRIPTION
**Pull request by nynevi**
- fixes body of posts in /posts page
- fixed the paid section body not being posted in put request
- fixed nth-child problem

### CODE REVIEW

File: src/editor/EditorCtrl.ts [hunk 1]
```diff
@@ -157,23 +155,39 @@ export default class EditorCtrl {
...
+            checkForCurrencyConversion();
+            const linebreak = $scope.draft.paidSectionLinebreak !== null ? $scope.draft.paidSectionLinebreak : 0;
+            setTimeout(() => {
+              scrollToLinebreak(undefined, linebreak);
+            }, 0);
           }
         }
 
+        const checkForCurrencyConversion = () => {
+          this.walletService.convertBCHtoUSD($scope.draft.paidSectionCost).then((currencies: ICurrencyConversion) => {
```
Comment (adrianbarwicki) [inline src/editor/EditorCtrl.ts]: You can use here async/await pattern.

File: src/editor/EditorCtrl.ts [hunk 2]
```diff
@@ -157,23 +155,39 @@ export default class EditorCtrl {
...
+            $scope.$apply(function () {
```
Comment (adrianbarwicki) [inline src/editor/EditorCtrl.ts]: We have the scopeService for that. You can firstly set the new param's values and then call the scopeService to apply the changes. In this way you can avoid the callback which is harder to read.

File: src/editor/services/EditorService.ts [hunk 3]
```diff
@@ -26,42 +26,40 @@ export default class EditorService {
...
     const bodyHtml = externalHtml ? externalHtml : editor.serialize().body.value;
-
-    let $bodyHtml = $(bodyHtml);
+    const $bodyHtml = $(bodyHtml);
 
     // the html is to replace the body in the editor
     let _fixedBody = '';
 
-    $bodyHtml = $bodyHtml.map(i => {
+    $bodyHtml.map(i => {
```
Comment (adrianbarwicki) [inline src/editor/services/EditorService.ts]: Please use an explicit name for the variable instead of i. `i` stands for index.

**Comment by adrianbarwicki** [comment]
Generally the changes are good. Few little formatting changes requested.

File: src/editor/services/EditorService.ts (hunk 3 above)
Comment (nynevi) [reply src/editor/services/EditorService.ts]: it is irrelevant

**Pull request merged**